src/client/Microsoft.Identity.Client/Http/HttpManager.cs

```diff
@@ -54,8 +73,20 @@ public async Task<HttpResponse> SendRequestAsync(
             X509Certificate2 bindingCertificate,
             Func<HttpRequestMessage, X509Certificate2, X509Chain, SslPolicyErrors, bool> validateServerCert,
             CancellationToken cancellationToken,
```

Comment:
nit: since you are changing this method signature anyway, I recommend moving CancellationToken to be the last parameter and to have a default value of `default`

Reply:
We used to have default value for the cancellation token and then we removed it to be passed back. What is the reason to have a default value for cancellation token? @bgavrilMS

Reply:
Convention. But I see your point - force MSAL developers to think about cancellation token. Thx.